package.json

```diff
@@ -18,7 +18,6 @@
     "@heroicons/vue": "^2.1.3",
     "axios": "^1.6.8",
     "pinia": "^2.1.7",
-    "portable-fetch": "^3.0.0",
```

Comment:
did you check that the API calls still work? AFAIK this was required.

FWIW, I think the vuln only affects server-side usage, and only when fetching from an untrusted server: https://github.com/advisories/GHSA-w7rc-rwvf-8q5r

Reply:
Yeah it still seems to work and I can't see that it is required, but I'll double check later once deployed